frontend/src/pages/AccountInfo.tsx

```diff
@@ -48,37 +48,35 @@ export default class AccountInfo extends React.Component<
 
 	render() {
 		return (
-			<div>
+			<div className="registerBody">
 				<div className="h1">Account Information</div>
```

Comment:
Instead of using the h1 style can we create a content body heading style, something like 'body-heading'. 

This will make changing either h1s or the content body headings easier in the future without CSS side affects we need to hunt for manually if we make CSS changes.

Reply:
Headings for all pages are now tagged with 'body-heading'